lib/dataset-tools/augmentation/replace_parameters.js

```diff
@@ -341,6 +341,14 @@ module.exports = class ParameterReplacer {
                     sentenceValue = sentenceValue.substring(match[0].length);
             }
         }
+
+        // hack for hotel domain: remove "hotel" in the end of names
+        if (arg.name === 'id' && programValue.endsWith(' hotel')) {
+            programValue = programValue.substring(0, programValue.length - ' hotel'.length);
+            if (coin(0.5, this._rng))
+                sentenceValue = sentenceValue.substring(0, sentenceValue.length - ' hotel'.length);
+        }
```

Comment:
Should probably do this only if the domain is hotel. Otherwise, searching for [Tokyo Hotel](https://en.wikipedia.org/wiki/Tokio_Hotel) won't work very well...
(Used to be all the rage for my female friends when I was in middle school!)

Reply:
Addressed